### ISSUE TITLE
Bayesian Deep Learning

### ISSUE THREAD

**Issue by howardyclo** [howardyclo/papernotes#69]
### Tutorials and Seminars
- [Andrew Gordon Wilson on Bayesian deep learning; Primer: Pavel Izmailov and Polina Kirichenko](https://www.youtube.com/watch?v=GXs9Pmp6IKQ). Broad Institute of MIT and Harvard. [[Slides](https://polkirichenko.github.io/MIA_talk.pdf)].
- [Bayesian ML/DL Playlist](https://www.youtube.com/channel/UC9KcwaZ9gSvcYNs7Jx3oNaQ/playlists) by [Bayesian Methods Research Group from Moscow, Russia](https://bayesgroup.ru/about/).

### PhD Thesis
- [Bayesian Method for Adaptive Models. David J.C. MacKay (1992)](https://thesis.library.caltech.edu/25/2/MacKay_djc_1992_revised_by_author.pdf)
- [Bayesian Learning for Neural Networks. Lecture Notes in Statistics. Neal, R. M. (1996)](https://sci-hub.tw/https://link.springer.com/book/10.1007%2F978-1-4612-0745-0)
- [Uncertainty in Deep Learning. Yarin Gal (2016)](http://mlg.eng.cam.ac.uk/yarin/blog_2248.html)

#### 2015
- [Weight Uncertainty in Neural Networks](https://arxiv.org/abs/1505.05424) by Charles Blundell. ICML 2015.

#### 2016
- [Dropout as a Bayesian Approximation: Representing Model Uncertainty in Deep Learning](https://arxiv.org/abs/1506.02142) by Yarin Gal. ICML 2016.

#### 2017
- [On Calibration of Modern Neural Networks](https://arxiv.org/pdf/1706.04599.pdf) by Chuan Guo et al. ICML 2017.
- [Simple and Scalable Predictive Uncertainty Estimation using Deep Ensembles](https://arxiv.org/pdf/1612.01474.pdf) by Lakshminarayanan et al. NIPS 2017.
- [What Uncertainties Do We Need in Bayesian Deep Learning for Computer Vision?](https://papers.nips.cc/paper/7141-what-uncertainties-do-we-need-in-bayesian-deep-learning-for-computer-vision.pdf) by Alex Kendall & Yarin Gal. NIPS 2017.

#### 2018
- [Multi-Task Learning Using Uncertainty to Weigh Losses for Scene Geometry and Semantics](http://openaccess.thecvf.com/content_cvpr_2018/papers/Kendall_Multi-Task_Learning_Using_CVPR_2018_paper.pdf) by Alex Kendall et al. CVPR 2018.
- [Uncertainty Estimates and Multi-Hypotheses Networks for Optical Flow](https://arxiv.org/abs/1802.07095) by Ilg et al. ECCV 2018.
- [Accurate Uncertainties for Deep Learning Using Calibrated Regression](http://proceedings.mlr.press/v80/kuleshov18a/kuleshov18a.pdf) by Volodymyr Kuleshov et al. ICML 2018.
- [Predictive Uncertainty Estimation via Prior Networks](https://arxiv.org/abs/1802.10501) by Andrey Malinin et al. NeurIPS 2018.

####  2019
- [Bias-Reduced Uncertainty Estimation for Deep Neural Classifiers](https://arxiv.org/abs/1805.08206v4) by Geifman et al. ICLR 2019.
- [Why ReLU networks yield high-confidence predictions far away from the training data and how to mitigate the problem](http://openaccess.thecvf.com/content_CVPR_2019/papers/Hein_Why_ReLU_Networks_Yield_High-Confidence_Predictions_Far_Away_From_the_CVPR_2019_paper.pdf) by Matthias Hein et al. CVPR 2019.
- [Noise Contrastive Priors for Functional Uncertainty](https://arxiv.org/abs/1807.09289) by Danijar Hafner et al. UAI 2019.
- [Sampling-free Uncertainty Estimation in Gated Recurrent Units with Applications to Normative Modeling in Neuroimaging](http://auai.org/uai2019/proceedings/papers/296.pdf) by Seong Jae Hwang et al. UAI 2019.
-  [Sampling-free Epistemic Uncertainty Estimation Using Approximated Variance Propagation](https://arxiv.org/abs/1908.00598) by Janis Postels et al. ICCV 2019.
- [Can you trust your model's uncertainty? Evaluating predictive uncertainty under dataset shift](http://papers.nips.cc/paper/9547-can-you-trust-your-models-uncertainty-evaluating-predictive-uncertainty-under-dataset-shift.pdf) by Yaniv Ovadia et al. NeurIPS 2019.
- [A Simple Baseline for Bayesian Uncertainty in Deep Learning](https://papers.nips.cc/paper/9472-a-simple-baseline-for-bayesian-uncertainty-in-deep-learning.pdf) by Maddox et al. NeurIPS 2019.
- [Single-Model Uncertainties for Deep Learning](http://papers.nips.cc/paper/8870-single-model-uncertainties-for-deep-learning.pdf) by Natasa Tagasovska & David Lopez-Paz. NeurIPS 2019.
- [Verified Uncertainty Calibration](http://papers.nips.cc/paper/8635-verified-uncertainty-calibration.pdf) by Ananya Kumar, Percy Liang, Tengyu Ma. NeurIPS 2019.

#### 2020
- [Pitfalls of In-Domain Uncertainty Estimation and Ensembling in Deep Learning](https://bayesgroup.ru/publications/2020-pitfalls-of-in-domain-uncertainty-estimation-and-ensembling-in-deep-learning/) by Arsenii Ashukha, +2, Dmitry Vetrov. ICLR 2020.
- [Scalable Uncertainty for Computer Vision with Functional Variational Inference](https://arxiv.org/abs/2003.03396) by Carvalho et al. CVPR 2020.

#### 2017
- [Stochastic Gradient Descent as Approximate Bayesian Inference](https://arxiv.org/abs/1704.04289) by Stephan Mandt et al. JMLR 2017.

#### 2018
- [Averaging Weights Leads to Wider Optima and Better Generalization](https://arxiv.org/abs/1803.05407) by Pavel Izmailov, +2, Dmitry Vetrov, Andrew Gordon Wilson. UAI 2018.

#### 2019
- [Understanding Priors in Bayesian Neural Networks at the Unit Level](http://proceedings.mlr.press/v97/vladimirova19a/vladimirova19a.pdf) by Vladimirova et al. ICML 2019.
- [The Deep Weight Prior](https://bayesgroup.ru/publications/2019-the-deep-weight-prior/) by Andrei Atanov, +2, Dmitry Vetrov, Max Welling. ICLR 2019.
- [Functional Variational Bayesian Neural Networks](https://arxiv.org/abs/1903.05779) by Sun, +2, Roger Grosse. ICLR 2019.
- [Subspace Inference for Bayesian Deep Learning](https://arxiv.org/abs/1907.07504) by Pavel Izmailov, +3,  Dmitry Vetrov, Andrew Gordon Wilson. UAI 2019.
- [Practical Deep Learning with Bayesian Principles](https://arxiv.org/abs/1906.02506) by Kazuki Osawa & Emtiyaz Khan. NeurIPS 2019.
- [Approximate Inference Turns Deep Networks into Gaussian Processes](https://papers.nips.cc/paper/8573-approximate-inference-turns-deep-networks-into-gaussian-processes.pdf) by Emtiyaz Khan et al. NeurIPS 2019.

#### 2020
- [Bayesian Deep Learning and a Probabilistic Perspective of Generalization](https://arxiv.org/pdf/2002.08791.pdf) by Andrew Gordon Wilson. 
- [Rethinking Parameter Counting in Deep Models: Effective Dimensionality Revisited](https://arxiv.org/abs/2003.02139) by Wesley J. Maddox, +1,  Andrew Gordon Wilson.
